watchos/PeopleInSpaceWatch/Podfile

```diff
@@ -0,0 +1,4 @@
+target 'PeopleInSpaceWatch WatchKit Extension' do
```

Comment:
This was the line of the Podfile that took me the most time to figure out. You have to set the target correctly otherwise the common framework isn't found in the UI code.